packages/datadog-plugin-next/test/next.config.js

```diff
@@ -1,5 +1,9 @@
 module.exports = {
   eslint: {
     ignoreDuringBuilds: true
+  },
+  output: 'standalone',
```

Comment:
Is this needed if it's configured programatically?

Reply:
This is what tells next to create the `server.js` file found under `.next/standalone`, so I feel it's needed, unless it should be configured programmatically instead.

Reply:
Gotcha, so it just adds additional files that we can then use in tests but otherwise the previous tests can also run fine?

Reply:
Exactly. Technically, Next complains that since it's enabled, the server shouldn't be started normally, but it is just a warning, and in my experience and testing, starts fine (and all previous tests still pass). However, I did spend some time seeing if it would be better to have two config files in a folder and swap out the contents of `next.config.js` conditionally, but it seemed messy. I can swap back to an approach like this if we want to truly distinguish between standalone tests and not